internal: usage tracking (#2318)
bentoml/BentoML: README.md

[<img src="https://raw.githubusercontent.com/bentoml/BentoML/main/docs/source/_static/img/bentoml-readme-header.jpeg" width="600px" margin-left="-5px">](https://github.com/bentoml/BentoML)

> <br>

Reviewer: This doesn't seem like it's doing much? Maybe add another one?

> # 🍱 BentoML: Unified Model Serving Framework  [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=BentoML:%20The%20Unified%20Model%20Serving%20Framework%20&url=https://github.com/bentoml&via=bentomlai&hashtags=mlops,bentoml)

Reviewer: ```suggestion
# 🍱 BentoML: The Unified Model Serving Framework  [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=BentoML:%20The%20Unified%20Model%20Serving%20Framework%20&url=https://github.com/bentoml&via=bentomlai&hashtags=mlops,bentoml)
```

### Usage Reporting ###

BentoML collects anonymous usage data that helps our team to improve the product.

> Only BentoML's internal API calls and CLI commands are being reported and we striped out any potentially 
> sensitive information as much as possible, we will never collect user code, model data, model name or stacktrace. 
> Here's the [code](./bentoml/_internal/utils/analytics/usage_stats.py) for usage tracking.
> You can opt-out of usage tracking by the `--do-not-track` CLI option:

Reviewer: ```suggestion
Only BentoML's internal API calls and CLI commands are being reported. We strip out as much potentially 
sensitive information as possible, and we will never collect user code, model data, model names, or stack traces.
Here's the [code](./bentoml/_internal/utils/analytics/usage_stats.py) for usage tracking.
You can opt-out of usage tracking by the `--do-not-track` CLI option:
```

### Usage Reporting ###

BentoML collects anonymous usage data that helps our team to improve the product.
Only BentoML's internal API calls and CLI commands are being reported and we striped out any potentially 
sensitive information as much as possible, we will never collect user code, model data, model name or stacktrace. 
Here's the [code](./bentoml/_internal/utils/analytics/usage_stats.py) for usage tracking.
You can opt-out of usage tracking by the `--do-not-track` CLI option:
```bash
bentoml [command] --do-not-track
```

Or by setting environment variable `BENTOML_DO_NOT_TRACK=True`:
```bash
export BENTOML_DO_NOT_TRACK=True
```

### License ###

> [Apache License 2.0](./LICENSE)

Reviewer 2: can we change this back, this doesn't work on the pypi page

### Usage Reporting ###

>  We rely heavily of

Reviewer 2: ```suggestion
```

### Usage Reporting ###

> By default, BentoML anonymously collect usage data, and here's the [code](./bentoml/_internal/utils/analytics/usage_stats.py) for it. We rely heavily of

Reviewer 2: BentoML collects anonymous usage data that helps our team to improve the product. Only BentoML's internal API calls and CLI commands are being reported and we striped out any potentially sensitive information as much as possible, we will never collect user code, model data, model name or stacktrace. Here's the [code] for usage tracking.

You can opt-out of usage tracking by the `--do-not-track` CLI option:
```bash
> bentoml [command] --do-not-track
```

Or by setting environment variable `BENTOML_DO_NOT_TRACK=True`:
```bash
> export BENTOML_DO_NOT_TRACK=True
```

Author: done
